core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy

```diff
@@ -57,4 +57,14 @@ class BintrayConfiguration {
         }
     }
 
+    private void deriveDefaultsFromProject(Project project) {
+        if (extension.versionAttributes.isEmpty()) {
+            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
+            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
+            if (bestPluginId != null) {
+                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
+                println "Using plugin identifier '" + extension.versionAttributes.get('gradle-plugins') + "' for gradle portal."
```

Comment:
How would you inform the user about the choice that has been made?
Am 02.07.2015 10:32 schrieb "Paul Blundell" <notifications@github.com>:

> In
> core/src/main/groovy/com/novoda/gradle/release/BintrayConfiguration.groovy
> <https://github.com/novoda/bintray-release/pull/59#discussion_r33756759>:
>
> > @@ -57,4 +57,14 @@ class BintrayConfiguration {
> >          }
> >      }
> >
> > +    private void deriveDefaultsFromProject(Project project) {
> > +        if (extension.versionAttributes.isEmpty()) {
> > +            def gradlePluginPropertyFinder = new GradlePluginPropertyFinder(project)
> > +            String bestPluginId = gradlePluginPropertyFinder.findBestGradlePluginId()
> > +            if (bestPluginId != null) {
> > +                extension.versionAttributes << ['gradle-plugins': "$bestPluginId:$extension.groupId:$extension.artifactId"]
> > +                println "Using plugin identifier '" + extension.versionAttributes.get('gradle-plugins') + "' for gradle portal."
>
> is this spammed on a users console when they run the command? it's not
> wise to spam :(
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/bintray-release/pull/59/files#r33756759>.
>